### PR TITLE
Sumex Customizations (Swiss Medical Invoices) (IP-496, IP-499)

### PR DESCRIPTION
**Pull request by denysvitali**
With this PR InvoicePlane gains the ability to generate some Swiss Medical Invoices (Sumex, MediData, SUVA invoices, it has many names!).
Read more about it [here](http://www.forum-datenaustausch.ch/) (only available in German and French) and [here](http://sumex1.net/) (English only). 
Unfortunately a "pdf generator server" is needed, since the Invoices can be generated only on Windows.

Once everything is properly setup (`SUMEX_URL=` in `ipconfig.php`, some settings tuned and an invoice group containing "sumex" is created) it is possible to generate the Invoices PDFs, and eventually the XML, if needed
screenshot from 2017-03-05 01-25-01

Please note that we _didn't fully followed the standard_ (we needed some customization to meet our clients requirements), but it shouldn't be difficult to adapt it to be fully standard-compatible.

To fix the invoice generation problem we've created a C# application that gets the XML via HTTPS, and returns a generated PDF. We'll probably open source this software too in the near future.

If you have some question about this, or want to implement this feature in your InvoicePlane installation we're ready to help you, it's kind of tricky, but if you fill in all the fields everything works properly 👍 

Enjoy! 🚀 

cc/ @MarcoScarfo

----

Additional fixes:

*     IP-496 - Add email address to client selects to differ in case of duplicate names

### CODE REVIEW

**Comment by Kovah** [comment]
Did you fixed any other parts of the application that may have an issue reference?

**Comment by Kovah** [comment]
screen shot 2017-03-05 at 10 35 08
Dafuq? 😳

**Comment by denysvitali** [comment]
LOL, I guess I mistakenly included my own "output" folder, which included a doc reference for every function in InvoicePlane. I'll fix it ASAP, sorry about that

**Comment by denysvitali** [comment]
screenshot from 2017-03-05 10-50-05
Seems a little bit more reasonable this number 😆

**Comment by Kovah** [comment]
Thanks!
could you also add a small introduction for Sumex and how to configure it for the [Wiki](https://github.com/InvoicePlane/Wiki)?

**Comment by denysvitali** [comment]
The last commit adds the stamps to the web invoice (they were removed with the theme thing I think).
I've created them from scratch, and included the svg, and png + @2x. It would be nice to have them language dependent though 

screenshot from 2017-03-05 22-25-43
screenshot from 2017-03-05 22-24-53

File: application/modules/guest/controllers/View.php [hunk 1]
```diff
@@ -70,6 +70,9 @@ public function invoice($invoice_url_key)
 
             $this->load->view('invoice_templates/public/' . $this->mdl_settings->setting('public_invoice_template') . '.php', $data);
         }
+        else{
+          echo '<h2>Invoice not found</h2>';
```
Comment (Kovah) [inline application/modules/guest/controllers/View.php]: Maybe use `trans()` here?

Comment (denysvitali) [inline application/modules/guest/controllers/View.php]: Yes, I agree. I'll fix ASAP

File: application/modules/setup/views/prerequisites.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
     <div class="install-panel">
 
         <h1 id="logo"><span>InvoicePlane</span></h1>
-
+        <?php var_dump($errors); ?>
```
Comment (denysvitali) [inline application/modules/setup/views/prerequisites.php]: This btw should be removed, I've missed it

**Pull request merged**